editor/src/components/shared/__snapshots__/project-components.spec.ts.snap

```diff
@@ -52,26 +52,7 @@ Array [
               "propertyElements": Array [],
             },
           },
-          "props": Array [
```

Comment:
are these changes expected?

Reply:
Yes, this is just a revert of last week's change which made even inserting with the A button and the Add Element popup using absolute positioning: https://github.com/concrete-utopia/utopia/commit/c4c81aeb1db424b42ccd82a8a7f1727cfe319769 (which is a followup to https://github.com/concrete-utopia/utopia/commit/6d99bce4830e6484937ac1c808693ee5ccc1bf22 )

So basically this PR is also a fix of a recent regression.